stable/ambassador/templates/crds.yaml

```diff
@@ -28,6 +28,57 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
+metadata:
+  name: consulresolvers.getambassador.io
```

Comment:
Can you add
```yaml
  labels:
    app.kubernetes.io/name: {{ include "ambassador.name" . }}
    helm.sh/chart: {{ include "ambassador.chart" . }}
    app.kubernetes.io/instance: {{ .Release.Name }}
    app.kubernetes.io/managed-by: {{ .Release.Service }}
  {{ if .Values.crds.keep }}
  annotations:
    "helm.sh/resource-policy": keep
  {{ end }}
```

So it matches the other CRDs we have :)

Reply:
Sorry, added.